### PR TITLE
Theme fixes and cleanup

### PR DESCRIPTION
**Pull request by Riksu9000**
A disabled state was added to the default button theme. Now Stopwatch, SetTime and SetDate match.
Arrow icons were added. These are used for dropdowns.
Arc colors were swapped.
Steps text was made lighter as it was too dark.
The default bar theme now has a background. This is visible on the firmware update screen and navigation.
Unused styles were removed and other small improvements in `lv_pinetime_theme.c`

This touches #953, but the dropdown still has some issues.

date
metronome
navigation
steps
update

### CODE REVIEW

**Comment by trman** [comment]
why did you swap the arc color ? 
to force your color taste?
the rest of the modification could related to button and drop down were good but 
but tying these with color change... is tasteless...

**Comment by Riksu9000** [comment]
It's common for the arc to fill clockwise and for the foreground to be brighter than the background.

**Comment by trman** [comment]
> It's common for an arc to fill clockwise and for the foreground to be brighter than the background.

no , it wholly depend from themes it not "common" : 
if theme is black  for foreground
or 
if theme is white for  foreground

ilke night and white theme on android....

**Comment by trman** [comment]
by the way , in my case , i like the actual theme rather that your's

**Comment by Riksu9000** [comment]
Let me rephrase that. Foreground should have more contrast. It should "pop" more. Also the InfiniTime theme is a dark theme, where background is dark and foreground is bright.

Screenshot from 2022-01-30 19-59-05

**Comment by trman** [comment]
thanks ! 
this explanation is more understandable !
now i understand this fix , because without it it felt like a color taste 'fix' !
